cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -90,7 +90,7 @@ spec:
         severity_level: critical
         storage_type: ceph
       expr: |
-        count(kube_pod_status_phase{pod=~"rook-ceph-mon-.*", phase=~"Running|running"}) by (namespace) < 2
+        count(kube_pod_status_phase{pod=~"rook-ceph-mon-.*", phase=~"Running|running"} == 1) by (namespace) < 2
```

Comment:
Doesn't the `< 2` already cover the `== 1` case?

Reply:
![image](https://user-images.githubusercontent.com/895234/139884979-58c26fa9-23cc-4ee4-8b39-fb9ee9b49ed8.png)

In the above example, we can see that if pod is in running phase value is `1` (at the right end). But if the pod is not running, the value is ZERO. So we have to collect / count pods which are in valid running state, that means the value is ONE.